subprojects/files/src/test/groovy/org/gradle/api/internal/file/collections/DirectoryFileTreeTest.groovy

```diff
@@ -259,7 +283,12 @@ class DefaultDirectoryWalkerTest extends AbstractProjectBuilderSpec {
 
         boolean assertExpectations() {
```

Comment:
I think the return type of this method should be `void`, so we don't need to return `true` from it.

Reply:
yeah, could have been a Spock bug back then, but now voids are okay as well. Done